### PR TITLE
LIN

### PR DESCRIPTION
**Pull request by RikardO-HM**
Information migrated from mx4-wiki with some
adjustments.
Add Deactivate LIN

### CODE REVIEW

File: docs/interfaces/mx4/lin.md [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+    master request.
+
+    data:
+    The LIN frame.
+    id | data | LIN checksum
+    id: With parity omit bit 7 and 6 for id without parity
+    LIN checksum: If it is a valid frame the checksum is here
+#### Overflow message
+    When a overflow occurs at any of the busses handling LIN a report of which
+    bus and how many overflows that have occured is sent to the Linux system.
```
Comment (albsod) [inline docs/interfaces/mx4/lin.md:195]:
```suggestion
    If an overflow occurs at any of the LIN buses, a report of the bus ID
    and the overflow count are sent to the Linux system.
```

**Comment by albsod** [comment]
I believe coprocessor is more commonly used than co-cpu, and PIC is even more opaque.

File: docs/interfaces/mx4/lin.md [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+### start of transmission (4 bytes)
+
+This is a byte sequence used to indicate the start of a frame.
+It consists of the 4 byte long sequence: ```0x7e 0x7e 0x7e 0xa7```
+
+### message length (1 byte)
+
+The length of message type (1 byte) + data (variable) + checksum (1 byte)
+The byte for message length is not included in message length.
+
```
[the remark this answers is not in the document]
Comment (RikardO-HM) [reply docs/interfaces/mx4/lin.md:48]: I agree most of it is from mx4-wiki. i will fix this.

File: docs/interfaces/mx4/lin.md [hunk 3]
```diff
@@ -0,0 +1,345 @@
...
+### message type (1 byte)
+
+Which type of operation that is requested.
+Values start at 1 and is sequentially increased.
```
[the remark this answers is not in the document]
Comment (RikardO-HM) [reply docs/interfaces/mx4/lin.md:52]: Yes, i am not sure how it works. The line is direct taken from mx4-wiki.

**Review by albsod — APPROVED** [review]

**Pull request merged**